### PR TITLE
feat: cosmos 'Unstake Input' modal

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

- add `views/Unstake.tsx` modal

## Notice

Before submitting a pull request, please make sure you have answered the following:

- [ ] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

#665

## Risk

N/A

## Testing

- Change useRenderForChains to render for ChainTypes.Ethereum
- Enable REACT_APP_FEATURE_COSMOS_INVESTOR feature toggle
- Click "Get Started" on "Cosmos Validator" staking row
- On the modal, click "Start Staking"
- Click "Unstake OSMO"

## Screenshots (if applicable)

image

### CODE REVIEW

File: src/plugins/cosmos/components/AmountToStake/AmountToStake.tsx [hunk 1]
```diff
@@ -14,6 +15,7 @@ type AmountToStakeProps = {
   }
 }
 export const AmountToStake = ({
```
Comment (gomesalexandre) [inline src/plugins/cosmos/components/AmountToStake/AmountToStake.tsx:17]: Obviously this now is handling both stake and unstake copies.
Happy to implement renaming suggestions.

File: src/plugins/cosmos/components/modals/Staking/views/Unstake.tsx [hunk 2]
```diff
@@ -0,0 +1,237 @@
...
+            <CText color='gray.500' my='25px'>
+              {translate('staking.itWillTake')}
+              <span> </span>
+              <Box as='span' fontWeight='semibold' color='white'>
+                14 {translate('common.days')}
+              </Box>
+              <span> </span>
+              {translate('staking.toUnlock')}
+            </CText>
+          </Box>
```
Comment (NeOMakinG) [inline src/plugins/cosmos/components/modals/Staking/views/Unstake.tsx:168]: This `Box` is useless I think so

File: src/plugins/cosmos/components/modals/Staking/views/Unstake.tsx [hunk 3]
```diff
@@ -0,0 +1,237 @@
...
+              lineHeight={1}
+              color='gray.500'
+              translation={['staking.assetStakingBalance', { assetName: asset.name }]}
+            />
+            <Amount.Crypto value={cryptoAmountStaked || ''} symbol={asset.symbol} />
+          </Flex>
+          <Divider />
+          <FormControl>
+            <AmountToStake
+              width='100%'
```
Comment (NeOMakinG) [inline src/plugins/cosmos/components/modals/Staking/views/Unstake.tsx:180]:
```suggestion
```

You should test without this one, looks useless as `FormControl` is just a div with `width: 100%` and `AmountToStake` is a flexbox, it's probably naturally 100%

If not, then just resolve it

**Review by theoboldfrazier — APPROVED** [review]
Small comment, but overall looks good.

File: src/assets/translations/messages-en.json [hunk 4]
```diff
@@ -168,7 +170,10 @@
     }
   },
   "staking": {
-    "staking": "Staking"
+    "staking": "Staking",
+    "itWillTake": "It will take",
+    "toUnlock": "to unlock your Osmo after you unstake. Staking rewards will need to withdrawn separately.",
```
Comment (0xdef1cafe) [inline src/assets/translations/messages-en.json:175]: translations like this should be one translation and template in the string for grammatical and UI formatting reasons

**Review by 0xdef1cafe — APPROVED** [review]

File: src/assets/translations/messages-en.json (hunk 4 above)
Comment (gomesalexandre) [reply src/assets/translations/messages-en.json:175]: How would we handle the span in this case? `XX days` should be bold, so AFAIK there is no way to do that with `react-polyglot`, or at least it's not documented. We would probably need something a la `dangerouslySetInnerHTML`

**Pull request merged**